boards/nxp/imx93_evk/doc/index.rst

```diff
@@ -108,6 +110,23 @@ Serial Port
 This board configuration uses a single serial communication channel with the
 CPU's UART2 for A55 core and M33 core.
 
+uSDHC (SD or eMMC Interface)
```

Comment:
Assuming this can't be enabled on the board by default (is this USDHC interface shared with Linux/Uboot?)- can we add some form of testcase definition in `tests/subsys/sd/sdmmc` to validate that the subsystem works on this board?

Reply:
Hi, @danieldegrasse, good catch,  currently we rely on uboot or linux to start A-core zephyr, so Zephyr can't use the same uSDHC controller with uboot or Linux, that is to say:
1) if board is booted from SD Card (which uses uSDHC2), so we only could use uSDHC1 for Zephyr `tests/subsys/sd/mmc` testing.
2) if board is booted from mmc (which uses uSDHC1), so we only could use uSDHC2 for Zephyr `tests/subsys/sd/sdmmc` testing

I will update this document and provides overlay for both uSDHC controllers to address this issues, thanks.